x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/formatted_field.tsx

```diff
@@ -270,7 +270,7 @@ const FormattedFieldValueComponent: React.FC<{
       />
     );
   } else if (
-    fieldName === SENTINEL_ONE_AGENT_ID_FIELD ||
+    fieldName === AGENT_STATUS_FIELD_NAME &&
```

Comment:
This is the change that ensures that we show the agent status agains agent.status field and not `observer.serial_number` field.

Reply:
Interesting! Could you explain what is the difference between fieldName and fieldFromBrowserField?.name ? I am wondering if we will ever meet the following condition. 
Thanks 🙇

Reply:
As I understand it on the surface `fieldName` is what tells the component which field name to show on the table. This may be the actual field name or may have been overridden by some other custom string. `fieldFromBrowserField` is what the actual field name. 

Not sure what you mean by ever meeting the condition? 

With some testing it was clear that the current condition is too loose and thus shows agent status component for conditions when `fieldName: observer.serial_number` and `fieldFromBrowserField.name: observer.serial_number` (in table tab view) when in reality it should show the sentinel agent status component when `fieldName: agent.status` and `fieldFromBrowserField.name: observer.serial_number` (when on overview tabs).

Reply:
Thank you for explanation 👍 
I hope you don't mind if I dig a bit deeper here :P 
Does this change actually applies to `Overview tab`?  This again will show `agent.status` healthy badge in `Table tab` which I think won't ever happen since normally we do not have agent.status field in events, or am I wrong?

Reply:
When we're looking at `observer.serial_number` it shows correctly a string 👍 
But If I am not wrong, the code change above would show a healthy badge in `Table tab` next to `agent.status` field, which I believe does not exist, thus this change is redundant. 
Were you able to reproduce a situation where we have `fieldName: agent.status` and `fieldFromBrowserField.name: observer.serial_number` ?

Reply:
We discussed this offline, the `fieldName: agent.status` and `fieldFromBrowserField.name: observer.serial_number` is met on the `Timelines Overview tab` - however not happening in Alert Overview tab. 

So the change makes sense. Thanks for doing this @ashokaditya :)